settings.gradle

```diff
@@ -1,12 +1,26 @@
 pluginManagement {
+    plugins {
+        id 'io.micronaut.build.shared.settings' version '5.3.4'
+    }
     repositories {
         gradlePluginPortal()
         mavenCentral()
     }
 }
 
 plugins {
-    id 'io.micronaut.build.shared.settings' version '5.3.4'
+    id 'io.micronaut.build.shared.settings'
+}
+
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+    }
+    versionCatalogs {
+        "mn" {
+            from("io.micronaut:micronaut-bom:${providers.gradleProperty("micronautVersion").get()}")
+        }
+    }
 }
 
```

Comment:
Instead of this, in other modules we have moved everything into `libs.versions.toml` and replaced

```
    testImplementation(mn.micronaut.http.server.netty)
```

With

```
    testImplementation(libs.micronaut.http.server.netty)
```

ie: https://github.com/micronaut-projects/micronaut-redis/pull/270

I think consistency between projects is probably a noble goal 👍

Reply:
Yes, I intended for us not to use the `mn`. I would like every project should follow as much as possible the [micronaut-project-template](https://github.com/micronaut-projects/micronaut-project-template).  However, @hollingsworthd used the `mn` approach which was used in Micronaut Data and I did not reverted that. 

@melix should we use the `mn` approach in project template?

Reply:
The advantage of using `mn` is we have lighter `libs.versions.toml` files.